topic-operator/src/test/java/io/strimzi/operator/topic/TopicOperatorIT.java

```diff
@@ -654,4 +658,68 @@ public void testReconcile(TestContext context) {
     // TODO: What happens if we create and then change labels to the resource predicate isn't matched any more
     //       What then happens if we change labels back?
 
+
+    @Test
+    public void testKafkaTopicCreatedWithOwnerRef(TestContext context) {
```

Comment:
Since this now test more than just plain creation I think a better name would be `testKafkaTopicWithOwnerRef()`